### DISCUSSION TITLE
How to forward CLI arguments to a command

### DISCUSSION THREAD

**Discussion by johhnry** [casey/just#1673] [Q&A] [answered]
Hi,

I would like to forward the remainder arguments when launching a command with Just:

```yaml
run:
  node main.js
```

```shell
$ just run --out file.txt --type d
```

This would run: `node main.js --out file.txt --type d`

Is it possible?

Thanks!

**Comment by laniakea64** [comment, marked as answer]
You can give the recipe a [variadic parameter](https://just.systems/man/en/chapter_38.html):
```
run *args:
  node main.js {{args}}
```

If your additional command-line arguments contain whitespace or other characters that are special to the shell, see https://just.systems/man/en/chapter_58.html

**Reply by johhnry** [reply]
Thanks @laniakea64 it works perfectly!
